### PR TITLE
add-publisher-wdr

### PR DESCRIPTION
**Pull request by jannispoltier**
Hi, I added the WDR (Westdeutscher Rundfunk) to the collection of German publishers and ran all tests and commands as instructed. 
There is just one issue I couldn't resolve as for some article summaries the crawler returns a `<p class="stand small">` element although I added an XPath to return `<p class="einleitung small">` and this element did exist in the given file. However, I couldn't come up with a fix for that problem. Maybe you have a suggestion :)

### CODE REVIEW

File: src/fundus/publishers/de/wdr.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+)
+
+
+class WDRParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = XPath(
+            "//article//p[starts-with(@class,'text') and not(position()=last())  and not(contains(text(), 'Quelle'))]"
+        )
+        _summary_selector = XPath("//article//p[starts-with(@class,'einleitung')]")
+        _subheadline_selector = XPath("//article//h2")
```
Comment (MaxDall) [inline src/fundus/publishers/de/wdr.py:22]:
```suggestion
        _subheadline_selector = XPath("//article//h2[@class='subtitle small']")
```

File: src/fundus/publishers/de/wdr.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        def authors(self) -> List[str]:
+            return generic_author_parsing(self.precomputed.meta.get("Author"))
+
+        @attribute
+        def publishing_date(self) -> Optional[datetime.datetime]:
+            return generic_date_parsing(self.precomputed.ld.bf_search("datePublished"))
+
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.ld.bf_search("name")
```
Comment (MaxDall) [inline src/fundus/publishers/de/wdr.py:43]:
```suggestion
            return self.precomputed.meta.get("og:title")
```

**Review by MaxDall — CHANGES_REQUESTED** [review]

**Comment by MaxDall** [comment]
Hey, @jannispoltier could you rerun `python -m scripts.generate_parser_test_files -p WDR -oj` and push the modified JSON file? You don't have to remove the old test case, just run the script with `-oj` flag :)

**Comment by jannispoltier** [comment]
> Hey, @jannispoltier could you rerun `python -m scripts.generate_parser_test_files -p WDR -oj` and push the modified JSON file? You don't have to remove the old test case, just run the script with `-oj` flag :)

Hi @MaxDall, I pushed the file and on my local machine the tests were now successful ☺️

**Review by MaxDall — APPROVED** [review]

**Review by MaxDall — APPROVED** [review]

**Review by addie9800 — APPROVED** [review]

**Pull request merged**